src/components/ProfilePage/UserEvents.tsx

```diff
@@ -0,0 +1,81 @@
+import Image from "next/image";
+import Link from "next/link";
+import * as Separator from "@radix-ui/react-separator";
+import TextIcon from "./TextIcon";
+import EventCard from "./ProfileEventCard";
+import RegisteredIcon from "../../../public/assets/icons/registered_events_icon.svg";
+import SavedIcon from "../../../public/assets/icons/bookmark_icon.svg";
+import BizBot from "../../../public/assets/bizbot_peeking.svg";
+import { BiztechEvent } from "@/types";
+import { isMobile } from "@/util/isMobile";
+import { useState, useEffect } from "react";
+
+interface UserEventsProps {
+  registeredEvents: BiztechEvent[];
+  savedEvents: BiztechEvent[];
+}
+
+export const UserEvents: React.FC<UserEventsProps> = ({
+  registeredEvents,
+  savedEvents,
+}) => {
+  const [isMobileDevice, setIsMobileDevice] = useState(false);
+  useEffect(() => {
+    const userAgent = navigator.userAgent;
+    setIsMobileDevice(isMobile(userAgent));
+  }, []);
+  return (
+    <div className="relative bg-profile-card-bg rounded-md p-6 w-full lg:w-[55%]">
+      <Image
+        src={BizBot}
+        alt="BizBot"
+        className="absolute top-[-150px] right-[-10px] h-[150px] hidden lg:block"
+      />
```

Comment:
I noticed this SVG is slow to load on the Vercel deployment, it comes out to 6.0 MB. Wondering if a PNG would be smaller and faster to load here?

Reply:
I've changed it to a png. It seems to be loading faster locally but I can't access the vercel deployment. Lmk if it worked.